ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/review/v1/review/clientlibs/site/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[core.forms.components.review.v1.runtime]"
+    dependencies="[core.forms.components.runtime.base,core.forms.components.container.v2.runtime]"/>
```

Comment:
review component should not depend on `core.forms.components.container.v2.runtime` its already fixed for other components